specification/monitor/resource-manager/Microsoft.Insights/stable/2023-03-11/dataCollectionRules_API.json

```diff
@@ -0,0 +1,1451 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Monitor Data Collection Rules API",
+    "description": "Provides operations for working with Azure Monitor data collection rules",
+    "version": "2023-03-11",
```

Comment:
your first commit has the files from 2022-06-01 for the Endpoints and the RuleAssociations , but not the Rules Can you please add that as well in a commit.

Reply:
Using open api would this automatically for you : 

![image](https://github.com/Azure/azure-rest-api-specs/assets/11067581/84c2528a-8b31-4e6c-b688-f5a27a4d8b50)

Reply:
I already did that in a second commit

Reply:
@raosuhas Any guidance on the next steps?